Tests/PublicAPISpec.swift

```diff
@@ -72,6 +72,19 @@ class PublicAPISpec: QuickSpec {
 
         describe("RInAppMessaging") {
 
+            it("will not crash if api methods are called prior to configure()") {
+                RInAppMessaging.deinitializeModule()
+                expect(RInAppMessaging.initializedModule).to(beNil())
+
+                let errorDelegate = ErrorDelegate()
+                RInAppMessaging.delegate = delegate
+                RInAppMessaging.errorDelegate = errorDelegate
```

Comment:
Should we call the delegate if an API is call without the module being initialized?

Reply:
Good idea. Done.